crmlogin/Contact.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/**
+Methods used to edit contacts. Used in actualContactEdit.php
+*/
+	class Contact{
+		
+
+		private $userInfo = array('firstname' => " ", 'lastname' => " ", 'email' => " ");
+		private $user; 
+		private $currentEmail;
+
+/**
+Instantiates the object by email
+*/
+		public static function GetByEmail($email){ 
+			include 'connection.php';
+			session_start();
+			global $user;
```

Comment:
Globals always bad... have the individual files check to see whether they're logged in or not. Keep this class focused on just the contact